astroid/brain/brain_collections.py

```diff
@@ -62,5 +58,16 @@ def __imul__(self, other): pass
         def __rmul__(self, other): pass'''
     return base_deque_class
 
+
+def _ordered_dict_mock():
+    base_ordered_dict_class = '''
+    class OrderedDict(dict):
+        def __reversed__(self): return self[::-1]
+    '''
+    if PY35:
```

Comment:
Nice! `move_to_end` was added in 3.2 however (https://docs.python.org/3.6/library/collections.html#collections.OrderedDict.move_to_end) so we at least want to include 3.4 (the earliest python3.x we support atm) in this transform.